docs/plasmapy_sphinx/extras.py

```diff
@@ -0,0 +1,109 @@
+"""
+This module contains extra (optional) sphinx setup parameters that are not need
+to run the `plasmapy_sphinx` extension but are useful for documenting.  See
```

Comment:
```suggestion
to run the `plasmapy_sphinx` extension but are useful for documentation.  See
```
Not sure if this suggestion is along with what you intended.  Otherwise I was wondering if there was another word that would follow "documenting".